pattern-1/bosh-release/create.sh

```diff
@@ -97,6 +97,9 @@ if [ ! -x "$(command -v bosh)" ]; then
 fi
 
 # move to the deployment directory
+if [ ! -d ../${deployment} ]; then
```

Comment:
Shall we add a .gitkeep file to the deployment folder? Then we can update the README with the structure of the BOSH release.